changelog.json

```
@@ -1,5 +1,9 @@
 {
   "releases": {
+    "3.3.10": [
+      "[Fixed] The last lines in diffs are visible when warning banners are displayed - #18237",
+      "[Fixed] Ignore a files parent directories on Windows. - #1203"
```

Same typo as in https://github.com/desktop/desktop/pull/18235#discussion_r1506414686

```suggestion
      "[Fixed] Ignore a file's parent directories on Windows. - #1203"
```